### ISSUE TITLE
Add !g trigger for google search

### ISSUE THREAD

**Issue by adamstirtan** [adamstirtan/ircbot#1]
Example URL: https://ajax.googleapis.com/ajax/services/search/web?v=1.0&q=Paris%20Hilton

Response:

{"responseData": {"results":[{"GsearchResultClass":"GwebSearch","unescapedUrl":"http://www.parishilton.com/","url":"http://www.parishilton.com/","visibleUrl":"www.parishilton.com","cacheUrl":"http://www.google.com/search?q\u003dcache:EgDCCgd54xQJ:www.parishilton.com","title":"\u003cb\u003eParis Hilton\u003c/b\u003e","titleNoFormatting":"Paris Hilton","content":"The official website of \u003cb\u003eParis Hilton\u003c/b\u003e, celebrity entrepreneur and pioneer of reality \ntelevision."},{"GsearchResultClass":"GwebSearch","unescapedUrl":"https://en.wikipedia.org/wiki/Paris_Hilton","url":"https://en.wikipedia.org/wiki/Paris_Hilton","visibleUrl":"en.wikipedia.org","cacheUrl":"http://www.google.com/search?q\u003dcache:HN2dDK2qxM4J:en.wikipedia.org","title":"\u003cb\u003eParis Hilton\u003c/b\u003e - Wikipedia, the free encyclopedia","titleNoFormatting":"Paris Hilton - Wikipedia, the free encyclopedia","content":"\u003cb\u003eParis\u003c/b\u003e Whitney \u003cb\u003eHilton\u003c/b\u003e (born February 17, 1981) is an American socialite, television \npersonality, model, actress, singer, DJ and author. She is the ..."},{"GsearchResultClass":"GwebSearch","unescapedUrl":"https://twitter.com/parishilton","url":"https://twitter.com/parishilton","visibleUrl":"twitter.com","cacheUrl":"http://www.google.com/search?q\u003dcache:6FskLjbnElkJ:twitter.com","title":"\u003cb\u003eParis Hilton\u003c/b\u003e (@\u003cb\u003eParisHilton\u003c/b\u003e) | Twitter","titleNoFormatting":"Paris Hilton (@ParisHilton) | Twitter","content":"25.5K tweets • 1421 photos/videos • 13.4M followers. \u0026quot;Had so much fun at my @\nfoam_diamonds Party! Loved rocking my jovanifashions dress ..."},{"GsearchResultClass":"GwebSearch","unescapedUrl":"https://instagram.com/parishilton/","url":"https://instagram.com/parishilton/","visibleUrl":"instagram.com","cacheUrl":"http://www.google.com/search?q\u003dcache:bFagY4AFMoEJ:instagram.com","title":"\u003cb\u003eParis Hilton\u003c/b\u003e (@\u003cb\u003eparishilton\u003c/b\u003e) • Instagram photos and videos","titleNoFormatting":"Paris Hilton (@parishilton) • Instagram photos and videos","content":"\u003cb\u003eParis Hilton\u003c/b\u003e. My new single \u0026quot;High Off My Love\u0026quot;: http://apple.co/1HiamjV\u200b \nSnapchat- RealParisHilton www.\u003cb\u003eParisHilton\u003c/b\u003e.com. 4,981 posts; 4.8m followers; \n1,449 ..."}],"cursor":{"resultCount":"10,300,000","pages":[{"start":"0","label":1},{"start":"4","label":2},{"start":"8","label":3},{"start":"12","label":4},{"start":"16","label":5},{"start":"20","label":6},{"start":"24","label":7},{"start":"28","label":8}],"estimatedResultCount":"10300000","currentPageIndex":0,"moreResultsUrl":"http://www.google.com/search?oe\u003dutf8\u0026ie\u003dutf8\u0026source\u003duds\u0026start\u003d0\u0026hl\u003den\u0026q\u003dParis+Hilton","searchResultTime":"0.29"}}, "responseDetails": null, "responseStatus": 200}
